### ISSUE TITLE
[Accessibility]: The value "pagination-select-current-page-0" of the 'for' attribute is not the 'id' of a valid <input> element

### ISSUE THREAD

**Issue by yashgarg1719** [IBM/carbon-components-angular#2197]
## Detailed description

> Describe in detail the issue you're having.

This issue is specific to `<ibm-pagination>` component where one of the `<label>` elements doe not have required attribute

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

`<ibm-pagination>`

> What did you expect to happen? What happened instead? What would you like to see changed?

The 'for' attribute must reference a non-empty, unique 'id' attribute of an <input> element

> What browser are you working in?

Firefox 101.0(64-bit)

> What version of the Carbon Design System are you using?

`carbon-components@10.5.0`
`carbon-components-angular@4.56.4`

> What WCAG 2.1 checkpoint does the issue violate?

IBM 1.3.1 Info and Relationships WCAG technique H44

## Additional information

* Screenshots or code

![image](https://user-images.githubusercontent.com/108860673/178191048-5cde4e34-260c-4db3-86bd-921ff7d7168e.png)

**Comment by Akshat55** [comment]
Cannot reproduce issue. Could you please post a code sandbox with issue? 

I'm not exactly sure why you're getting this error because the `for` value is the same as that of the `id` for select below.

Can you make sure your plugin is up to date & you're using the latest deployment for the rule set?
![image](https://user-images.githubusercontent.com/38994122/191165023-11016400-7564-44b7-a22f-464e9e5be2ef.png)
